tls/s2n_handshake_io.c

```diff
@@ -226,8 +226,10 @@ static int s2n_conn_update_handshake_hashes(struct s2n_connection *conn, struct
 {
     GUARD(s2n_hash_update(&conn->handshake.md5, data->data, data->size));
     GUARD(s2n_hash_update(&conn->handshake.sha1, data->data, data->size));
+    GUARD(s2n_hash_update(&conn->handshake.sha224, data->data, data->size));
```

Comment:
I wonder what keeping _all_ the hashes does to our handshake performance :) . In the short term this seems like the right decision(we don't know what the clients are using for sigalgs).